src/ReactWrapper.jsx

```diff
@@ -224,13 +225,14 @@ export default class ReactWrapper {
    * NOTE: can only be called on a wrapper instance that is also the root instance.
    *
    * @param {Object} state to merge
+   * @param {Function} cb - callback function
    * @returns {ReactWrapper}
    */
-  setState(state) {
+  setState(state, cb) {
     if (this.root !== this) {
       throw new Error('ReactWrapper::setState() can only be called on the root');
     }
-    this.instance().setState(state);
+    this.instance().setState(state, cb);
```

Comment:
Awesome - it would still be good to add tests for that behavior, in case a future version of react changes it